iam/x509-identity-mgmt/js/config/default.conf

```diff
@@ -312,3 +312,98 @@ logger.file.max:string=7d
 logger.file.size:string=10m
 
 # ******************************************************************************
+
+# =======================================
+# kafka consumer - General configurations
+# =======================================
+
+# ---------------------------
+# X509IDMGMT_KAFKA_TOPIC_ACKS
+# ---------------------------
+kafka.topic.acks:integer=-1
+
+# ----------------------------------------
+# X509IDMGMT_KAFKA_TOPIC_AUTO_OFFSET_RESET
+# ----------------------------------------
+kafka.topic.auto.offset.reset:string=latest
+
+# -----------------------------------
+# X509IDMGMT_KAFKA_CONSUMER_CLIENT_ID
+# -----------------------------------
+kafka.consumer.client.id:string=${X509IDMGMT_APP_HOSTNAME:-x509-identity-mgmt}
+
+# ----------------------------------
+# X509IDMGMT_KAFKA_CONSUMER_GROUP_ID
+# ----------------------------------
+kafka.consumer.group.id:string=x509-identity-mgmt-group
```

Comment:
As the group id is always the same, if you have several consumers only one of them will have this device in cache, is that what you really expect?
If the user wants to associate a device with a certificate, but you have not received any event to create that device in kafka, will you search for that device in the device-manager?

Reply:
1) The "cache" (in database) is shared by all instances of the _x509-identity-mgmt_ service, it is enough that only one _Kafka Consumer_ saves the device in the _x509-identity-mgmt_ database for all other instances of this service have access to that data. So I have no problem maintaining the same GroupId for all instances. Did I make a mistake?

2)  Yes, that's right!

Reply:
Ok, you right =)
There is no problem in here.